release-notes/opensearch-index-management-dashboards-plugin.release-notes-1.3.6.0.md

```diff
@@ -0,0 +1,9 @@
+## Version 1.3.6.0 2022-10-06
+Compatible with OpenSearchDashboards 1.3.0
```

Comment:
1.3.6?

Reply:
That's what they are telling me, see thread in slack